locking/models.py

```diff
@@ -71,7 +71,8 @@ def is_locked(self):
         Works by calculating if the last lock (self.locked_at) has timed out or not.
         """
         if isinstance(self.locked_at, datetime):
-            if (datetime.today() - self.locked_at).seconds < LOCK_TIMEOUT:
+            timedelta_since_lock = datetime.today() - self.locked_at
+            if timedelta_since_lock.days == 0 and timedelta_since_lock.seconds < LOCK_TIMEOUT:
```

Comment:
What about using [`total_seconds()`](https://docs.python.org/2/library/datetime.html#datetime.timedelta.total_seconds)?
```suggestion
            if timedelta_since_lock.total_seconds() < LOCK_TIMEOUT:
```

Reply:
oh, yeah, that would work!